dartagnan/src/main/java/com/dat3m/dartagnan/wmm/definition/Scopes/PTXScopes.java

```diff
@@ -0,0 +1,23 @@
+package com.dat3m.dartagnan.wmm.definition.Scopes;
+
+import com.dat3m.dartagnan.program.event.Tag;
+import com.dat3m.dartagnan.wmm.Definition;
+import com.dat3m.dartagnan.wmm.Relation;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class PTXScopes extends Definition {
+    private final ArrayList<String> scopes = new ArrayList<>(
```

Comment:
Why do you need this? You can simply pass the list to `v.visitSameScope`

Reply:
Accepted